### PR TITLE
Minor corrections on ROM and RAM id's for consistency

### PR DESCRIPTION
**Pull request by jlautenbag**
I noticed some inconsistencies in the machine xml files. Manuel invited me to create a pull request for this.

### CODE REVIEW

File: share/machines/Toshiba_HX-20E.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
     <primary slot="3">
```
Comment (MBilderbeek) [inline share/machines/Toshiba_HX-20E.xml:94]: Isn't it more logical to name the lower RAM "Main RAM 1" and the upper part "Main RAM 2"? Or is it the same in other configurations?
Perhaps we should change it the other way around, but then more standardized, like "Main RAM Lower part", "Main RAM Upper part"?
What do you think?

Comment (jlautenbag) [reply share/machines/Toshiba_HX-20E.xml:94]: Well, as you guessed, other configurations are like this. I had to look twice before making the change this way. :-)

"1" and "2" are very arbitrary indeed. I like the "Lower part" and "Upper part" better.

Comment (MBilderbeek) [reply share/machines/Toshiba_HX-20E.xml:94]: Thanks for the updates after our discussion. I think this is addressed now.

**Comment by MBilderbeek** [comment]
Thanks!

**Pull request merged**